.gitignore

```diff
@@ -3,3 +3,4 @@ composer.lock
 phpunit.xml
 vendor
 .php_cs.cache
+.idea
```

Comment:
This is not related, please revert & set it ignored in your local global gitignore.